reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt

```diff
@@ -250,18 +251,23 @@ class FreemarkerTemplateProcessor(
 
         /**
          * Merge the [ResolvedLicense]s of multiple [models] and filter them using [licenseView]. [Omits][omitExcluded]
-         * excluded packages, licenses, and copyrights by default. The returned list is sorted by license identifier.
+         * excluded packages, licenses, and copyrights by default. [Undefined][omitNotPresent] licenses can be filtered
+         * out optionally. The returned list is sorted by license identifier.
          */
+        @Suppress("MemberVisibilityCanBePrivate")
         @JvmOverloads
         fun mergeLicenses(
             models: Collection<PackageModel>,
             licenseView: LicenseView = LicenseView.ALL,
+            omitNotPresent: Boolean = false,
             omitExcluded: Boolean = true
         ): List<ResolvedLicense> =
             mergeResolvedLicenses(
-                models.filter { !omitExcluded || !it.excluded }.flatMap {
-                    val licenses = it.license.filter(licenseView).licenses
-                    if (omitExcluded) licenses.filterExcluded() else licenses
+                models.filter { !omitExcluded || !it.excluded }.flatMap { model ->
+                    val licenses = model.license.filter(licenseView).licenses
+                    val filteredLicenses = if (omitExcluded) licenses.filterExcluded() else licenses
+                    if (omitNotPresent) filteredLicenses.filter(this::isLicensePresent)
+                    else filteredLicenses
```

Comment:
I think the above two statements look nicer if they were inlined. Maybe consider adding  a new line for readability.

Reply:
I added a blank line before the two filter statements and packed the last statement into a single line.